.gitignore

```diff
@@ -1,25 +1,39 @@
+#generic development files
+www/index.html
+composer.lock
```

Comment:
Ignoring composer.lock does not seem like a good idea. Official Composer documentation encourage committing this file.

> Committing this file to VC is important because it will cause anyone who sets up the project to use the exact same versions of the dependencies that you are using.